tests/driver/runner_test.py

```diff
@@ -4824,54 +3810,33 @@ async def test_search_after_with_pit(self, es):
             "with-point-in-time-from": pit_op,
             "pages": "all",
             "results-per-page": 2,
-            "body": {
-                "sort": [{"timestamp": "asc", "tie_breaker_id": "asc"}],
-                "query": {"match-all": {}}
```

Comment:
Should this be `match_all`?

Reply:
Yes, it should be.

Reply:
Thanks, done in https://github.com/elastic/rally/pull/1281/commits/e57e08dcc9ec8aeade701e33cd13c0cc8594889e